src/components/content.js

```diff
@@ -698,15 +699,25 @@ class Content extends React.Component {
     if (this.props.readOnly) return
     if (!this.isInEditor(event.target)) return
 
-    event.preventDefault()
+    const handleData = (data) => {
```

Comment:
I understand that handling the paste event in a async way for IE 11 and synchronously for other browsers increases the complexity and makes debugging more difficult. But I don't know of any other workaround for this. 

That  logic with the temporary cloned and focused contenteditable element does work and as far as I know, there is no interference with react as the native paste action happens directly on the next tick after the onPaste event fires.